### ISSUE TITLE
[Enhancement] Refactor how line-height is used

### ISSUE THREAD

**Issue by jkaltoft** [kirbydesign/designsystem#3579]
- Line height declaration values should be unitless
- Line height is a ratio relative to a given element's font size
- We should be able to create the system's typography without many `line-height`-declarations
- Ideally the UX/design people should decide on and define a limited number of font size/line height ratios
 
-----

This issue is based on the findings of research https://github.com/kirbydesign/designsystem/issues/3568

Read more:
- [Success Criterion 1.4.4 Resize Text](https://www.w3.org/TR/WCAG22/#resize-text)
- [Understanding SC 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)
- About lenght units: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/length) and [W3C](https://www.w3.org/TR/css-values-4/#lengths)
